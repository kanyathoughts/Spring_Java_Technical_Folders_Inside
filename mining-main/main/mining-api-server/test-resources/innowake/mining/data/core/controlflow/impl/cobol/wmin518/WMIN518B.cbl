       IDENTIFICATION DIVISION.
       PROGRAM-ID. WMIN518B.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       PROCEDURE DIVISION.
           PERFORM TESTPARA
           GOBACK.
       TESTPARA.
            RETURN SORT-FILE RECORD INTO  OUTPUT-RECORD
            AT END
            DISPLAY 'A'
            NOT AT END
            DISPLAY 'B'
            END-RETURN.
       UNREACHABLE.
            DISPLAY 'UNREACHABLE'
