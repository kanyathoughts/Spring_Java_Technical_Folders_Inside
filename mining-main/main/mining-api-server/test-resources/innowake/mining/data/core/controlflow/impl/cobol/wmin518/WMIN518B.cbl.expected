#5 ProcedureDivision
 #6 CobolPerformStmt TESTPARA, FROM={TestEntryPoint}, TO={#9}
  #7 CobolFieldReference
 #8 CobolGoBackStmt, FROM={#15, #19}, TO={TestReturnPoint}
 #9 CobolLabelStmt TESTPARA , FROM={#6}, TO={#10}
  #10 CobolReturnStmt <branch stmt>, Label: RETURN, FROM={#9}, TO={#15(AT END), #19(NOT AT END)}
   #13 CobolFieldReference
   #14 CobolAtEndBlock
    #15 CobolDisplayStmt 'A', FROM={#10(AT END)}, TO={#8}
   #18 CobolAtEndBlock
    #19 CobolDisplayStmt 'B', FROM={#10(NOT AT END)}, TO={#8}
 #22 CobolLabelStmt UNREACHABLE 
  #23 CobolDisplayStmt 'UNREACHABLE'
